src/main/java/ui/components/pickers/MilestonePickerDialog.java

```diff
@@ -126,14 +128,30 @@ private void initUI() {
         assignedMilestoneBox = createAssignedMilestoneBox();
         matchingMilestonesBox = createMatchingMilestonesBox();
         inputField = createInputField();
+        ScrollPane allMilestoneScrollPane = createAllMilestonesScrollPane();
 
-        milestoneDialogBox.getChildren().addAll(assignedMilestoneBox, inputField, matchingMilestonesBox);
+        milestoneDialogBox.setSpacing(5);
+        milestoneDialogBox.getChildren().addAll(assignedMilestoneBox, inputField,
+                new Label(TITLE_RESULT), matchingMilestonesBox, new Label(TITLE_ALL), allMilestoneScrollPane);
```

Comment:
Indentation can be improved a bit here, to show that all these nodes are siblings.